crates/rooch-framework/sources/genesis.move

```diff
@@ -0,0 +1,30 @@
+module rooch_framework::genesis {
+
+    use moveos_std::storage_context::StorageContext;
+    use rooch_framework::auth_validator_registry;
+
+    const E_GENESIS_INIT: u64 = 1;
+
+    fun init(ctx: &mut StorageContext, genesis_account: &signer){
+        auth_validator_registry::genesis_init(ctx, genesis_account);
+        
+        //SCHEME_ED25519: u64 = 0;
+        let id = auth_validator_registry::register_internal<rooch_framework::ed25519_validator::Ed25519Validator>(ctx);
+        assert!(id == 0, std::error::internal(E_GENESIS_INIT));
```

Comment:
Considering extracting the validator schema in the auth_validator directory in the framework for a unified definition, so that can directly use the reuse definition instead of directly using numbers?